### ISSUE TITLE
open_with menu disappears along with input latency when previewing images

### ISSUE THREAD

**Issue by beanaroo** [ranger/ranger#1774]
#### Runtime Environment
- Operating system and version: Arch Linux - latest
- Terminal emulator and version: Kitty 
- Python version: 3.8.0
- Ranger version/commit: `4ff19f6`
- Locale: en_NZ

#### Current Behavior
Open_with menu takes a while to show and then disappears again.
Input latency apparent (for open_with only)

Please see below screencast:
![ranger-1774](https://user-images.githubusercontent.com/1748909/70369886-abcbd200-1924-11ea-8c56-888448d8358c.gif)

(Border problem reported as #1773, not sure if related)

#### Expected Behavior
Persisting open_with menu with no input latency

#### Possible Solutions
Disable image preview

#### Steps to reproduce
1. Enable image preview (kitty)
2. Present the open_with menu

**Comment by avalonv** [comment]
Also experiencing this issue with kitty

**Comment by perehodit** [comment]
The problem is related to `set preview_images_method`, when I changed from `kitty` to `ueberzug` all the problem was solved.

**Comment by toonn** [comment]
Yes, this problem is specific to the Kitty previewing method. Not sure why yet, would be appreciated if someone can dig into this.

**Comment by dgshanee** [comment]
I looked into the code, and it's the draw_possible_programs function that is causing the latency. Working on a solution now.

**Comment by toonn** [comment]
The fix was reverted in #3106 so this needs to be reopened.
